spec/ship_spec.rb

```diff
@@ -0,0 +1,60 @@
+require './lib/ship'
+require 'rspec'
+
+describe Ship do
+  before :each do
+    @cruiser = Ship.new("Cruiser", 3)
+  end
+
+  it "the ship exists" do
+    expect(@cruiser).to be_instance_of(Ship)
+  end
+
+  it "has a name" do
+    expect(@cruiser.name).to eq("Cruiser")
+  end
+
+  it "has a length" do
+    expect(@cruiser.length).to eq(3)
+  end
+
+  it "can have a different length" do
+    cruiser = Ship.new("Cruiser", 2)
+    expect(cruiser.health).to eq(2)
+  end
+
+  it "has health" do
+    expect(@cruiser.health).to eq(3)
+  end
+
+  it "can have different health" do
```

Comment:
Lets combine this test and the different length test into one test. 
Soo...

`cruiser_1 = Ship.new("Cruiser", 2)

expect(cruiser_1.length).to eq(2)
expect(cruiser_1.health).to eq(2)`